src/components/ArticleToc.js

```diff
@@ -14,7 +14,7 @@ export const ArticleToc = ({ items, currentAnchor, onSelectItem, close, closeAll
 
     if (item && item.title) {
       onSelectItem(item)
-      close()
+      closeAll()
```

Comment:
Little (unrelated) bug I found along the way, a bonus fix

Reply:
I don't advice doing "drive-by" fixes like that. If you do, make sure to provide steps to reproduce the issue that this is fixing.

The preferred way is to file a task describing the issue and link a one-line PR. Yes it takes longer, but it is so much more safer.

Reply:
Makes sense

> to reproduce the issue that this is fixing.

Issue was that when if you openned `ArticleToc` through the article menu, selecting a section there would route to correct section but the screen you saw was the article menu. This was because not all popups were closing, so this was [introduced recently](https://github.com/wikimedia/wikipedia-kaios/commit/62a2ef6c5dc7ee4ba47d463b6872f7d1d6632e92#diff-4cddd38dee83bd40943da8843dff58db) when we stacked popups from the article menu.

Will make sure to file a task next time.